### PR TITLE
Features Page

### PR DESCRIPTION
**Pull request by Irrreny**
Here is my try to make a 2nd page "Features3", and also i added responsiveness
image
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -143,6 +148,67 @@
...
+            background-color: white;
+            border: 15px solid var(--darkblue);
+            text-align: center;
+            color: #737373;
+        }
+        .grid-item-header {
+            display: flex;
+            gap: 15px;
+            justify-content: center;
+            padding: 35px 35px 0 35px;
```
Comment (emma13z) [inline index.html:176]: Just curious, are these the right padding dimensions? what is the purpose of 0?

File: index.html [hunk 2]
```diff
@@ -143,6 +148,67 @@
...
+        }
+        .grid-item-text {
+            padding: 15px 40px 35px 40px;
+            margin: 0;
+            color: #737373;
```
Comment (emma13z) [inline index.html:181]: As I see in Figma, the text in paragraphs is not centred. It is left-aligned.

File: index.html [hunk 3]
```diff
@@ -173,5 +239,56 @@ <h4 style="font-weight: lighter;">We know how large objects will act, but things
             </div>
             </div>
         </section>
+
+        <section class="why-us">
+        <div class="why-choose-us">
+            <h2>WHY CHOOSE US</h2>
```
Comment (emma13z) [inline index.html:245]: I don't see h2 and the paragraph below in your css. I think you need to give classes to them and style them. They don't look like in Figma. They are supposed to have different weight, line-height, etc. Check it out :)

**Review by emma13z — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Irrreny) [reply index.html:176]: I didn't put 35px on the bottom cause the item has the gap of 15px.
image

Comment (Irrreny) [reply index.html:176]: well, if i remove the padding, it looks like this :)
image

So, i just saw how it looks and kept the padding :)

Comment (emma13z) [reply index.html:176]: Hmm, I see you didn't add padding to your "grid-item". Never mind then, I thought it was in both parts :)

**Comment by Irrreny** [comment]
@emma13z Thanks for your comments :) i really didn't notice some things and you did 👍

File: index.html (hunk 1 above)
Comment (IvanCeban) [reply index.html:190]: Emma is right. The padding should be set to the parent. So your grid-item would have `padding: 35px 40px;`
children of grid-item should not have paddings. It should work.

File: index.html [hunk 4]
```diff
@@ -173,5 +262,56 @@ <h4 style="font-weight: lighter;">We know how large objects will act, but things
...
+        </div>
+        <div class="grid-content">
+            <div class="grid-item">
+                <div class="grid-item-header">
+                <img class="grid-icons" src="img/icons/briefcase.svg" alt="briefcase icon">
+                <h5 class="grid-header-text">Digital Marketing</h5>
+                </div>
+                <p class="grid-item-text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+            </div>
+            <div class="grid-item">
```
Comment (IvanCeban) [inline index.html:279]: I suggest you to rethink the grid-item namings.
Let's say what kind of data is that? If you take a closer look at the rest of the design you will notice that there are more of these cards.

So I suggest you to give each of the cards class card and use it for all sections in future.
The parent, the grid one contains cards. So you could give it a class called cards or cards-grid if you want to specify word "grid" there.
Inside each card you have "card header" and "card content". Here are the class names for them: card__header and card__content

Inside header you have icon and title.
So header__icn and header__title sound logical for their class names.

Basically you've chosen content agnostic class names as well. And you could use them for future elements. But "grid-item" doesn't tell us what kind of element is there. But "card" tells us that the element is a card and will look like a card. (it's a best practice to call cards elements like these)

File: index.html [hunk 5]
```diff
@@ -173,5 +262,56 @@ <h4 style="font-weight: lighter;">We know how large objects will act, but things
...
+                <div class="grid-item-header">
+                    <img class="grid-icons" src="img/icons/shop.svg" alt="Store-shop icon">
+                    <h5 class="grid-header-text">Digital Marketing</h5>
+                    </div>
+                    <p class="grid-item-text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+            </div>
+            <div class="grid-item">
+                <div class="grid-item-header">
+                <img class="grid-icons" src="img/icons/notebook.svg" alt="Store-shop icon">
```
Comment (IvanCeban) [inline index.html:288]: don't forget to edit lat text for each image

File: index.html [hunk 6]
```diff
@@ -173,5 +262,56 @@ <h4 style="font-weight: lighter;">We know how large objects will act, but things
             </div>
             </div>
         </section>
+
+        <section class="why-us">
+        <div class="why-choose-us">
+            <h2 class="why-choose-us-head">WHY CHOOSE US</h2>
```
Comment (IvanCeban) [inline index.html:268]: use CSS to make the text uppercase. Don't write it in uppercase in HTML as it is not a flexible solution.

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Good job! A couple of improvement suggestions

File: index.html [hunk 7]
```diff
@@ -173,5 +263,56 @@ <h4 style="font-weight: lighter;">We know how large objects will act, but things
             </div>
             </div>
         </section>
+
+        <section class="why-us">
+        <div class="why-choose-us">
+            <h2 class="why-choose-us-head">why choose us</h2>
```
Comment (IvanCeban) [inline index.html:269]: take in consideration that same title and subtitle styles are on all sections on the page. So you will have to invent new class names. 
Check out my message in general channel in slack

File: index.html [hunk 8]
```diff
@@ -173,5 +263,56 @@ <h4 style="font-weight: lighter;">We know how large objects will act, but things
             </div>
             </div>
         </section>
+
+        <section class="why-us">
+        <div class="why-choose-us">
```
Comment (IvanCeban) [inline index.html:268]: same as with title. If you want this wrapper over your title and subtitle, it's fine. But this wrapper will be the same in all sections. 
So `section-header` sounds a logic and meaningful class name for it. And you could use same class name in all sections

File: index.html [hunk 9]
```diff
@@ -143,6 +149,90 @@
...
+        }
+        .grid-cards {
+            display: grid;
+            grid-template-columns: repeat(3,328px);
+            background-color: var(--darkblue);
+            justify-content: center;
+        }
+        .card {
+            background-color: white;
+            border: 15px solid var(--darkblue);
```
Comment (IvanCeban) [inline index.html:183]: good for using css variables!

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Great naming improvements!
Still some stuff to improve left ;)

**Comment by Irrreny** [comment]
I hope now it's all fixed :)
But actually i start better understand the naming process.

**Review by sdmnc — APPROVED** [review]
so many comments and now approves :-)

**Review by IvanCeban — APPROVED** [review]
Well done!

**Comment by IvanCeban** [comment]
Don't forget to merge this PR in order to avoid conflicts with footer PR
